project/AppDependencies.scala

```diff
@@ -9,7 +9,7 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"       %% s"bootstrap-frontend-$playVersion" % "5.20.0",
+    "uk.gov.hmrc"       %% s"bootstrap-frontend-$playVersion" % "6.3.0",
```

Comment:
Can bootstrap go up to 7.12.0 in this ticket? Or is there a butterfly effect beyond scope?

Reply:
not in this ticket. that change should not be there it slept through from another ticket